src/components/App/App.css

```diff
@@ -50,6 +50,16 @@ main {
   cursor: pointer;
 }
 
+@media screen and (max-height: 750px) {
+    .sidebar-nav {
+        margin-top: 15px;
+        position: relative;
+    }
+    .sidebar {
+        overflow: auto;
```

Comment:
```suggestion
        overflow-y: auto;
```
? Just to make sure..

Reply:
@moshfeu It wasn't there before, I have just added it. Thanks man... Added. This can be now merged.